src/frontend/components/SøknadsSteg/VelgBarn/useVelgBarn.tsx

```diff
@@ -31,6 +35,17 @@ export const useVelgBarn = (): {
         settBarnForRoutes(barnSomSkalVæreMed);
     }, [barnSomSkalVæreMed]);
 
+    useEffect(() => {
```

Comment:
Bare et forslag, men hadde det vært en ide å flytte fjern barn-funksjonen inn i denne hooken, også bare fjerne barnet på en linje på lik linje som vi fjerne det fra søknaden? 

Bare er forslag fordi det føles litt mer hassle å følge hele flyten når logikken for fjern barn ligger litt spredd

Reply:
Ja, enig, det blir bedre sånn

Reply:
Nice :D